client/material/tooltip.tsx

```diff
@@ -146,6 +156,10 @@ export function Tooltip({
   const [mouseAnchorElem, setMouseAnchorElem] = useState<HTMLElement>()
   const [focusAnchorElem, setFocusAnchorElem] = useState<HTMLElement>()
   const anchorElem = focusAnchorElem ?? mouseAnchorElem
+  // NOTE(2Pac): We store the tooltip element here to indicate the mousing over the tooltip content
+  // so we can keep the tooltip open in case it's interactive.
+  const [tooltipElem, setTooltipElem] = useState<HTMLElement>()
```

Comment:
if you're just going to use this value as a boolean, then it seems better to just set a boolean instead of storing this